.github/workflows/labbox-ephys-install-tests.yml

```diff
@@ -0,0 +1,35 @@
+name: labbox-ephys-install-tests
+on:
+  pull_request:
+    types: [opened, synchronize]
```

Comment:
Trigger this on a PR, new commits on the PR's branch, or pushes to the master branch.